pensieve/statistics.py

```diff
@@ -111,6 +111,10 @@ class BootstrapMean(Statistic):
     def transform(self, df: DataFrame, metric: str) -> "StatisticResultCollection":
         stats_results = StatisticResultCollection([])
 
+        branch_list = df.branch.unique()
+        if self.ref_branch_label not in branch_list:
+            return stats_results
```

Comment:
Some experiments were failing since either the dataframe was empty or the reference branch wasn't in the dataframe.

Reply:
We should log a warning here.

Does calling .unique() before `in` end up doing less work?

We also need to perform this check in every other Statistic; is there a DRY way to do that?

Reply:
I added logging a warning. 
I guess `unique` is not really necessary here.

I moved the check into the abstract `Statistics` class.